File: M3 Intermediate DSA/src/L4_Introduction_to_Arrays/Assignment3.java
package L4_Introduction_to_Arrays;

public class Assignment3 {

  public int[] solve(int[] A, int B, int C) {
    int index = B;
    int jIndex = C;

    while (index < jIndex) {
      int tmp = A[index];
      A[index] = A[jIndex];
      A[jIndex] = tmp;
      index++;
      jIndex--;
    }

    return A;
  }

}

File: M3_Intermediate_DSA/L2_Introduction_to_Arrays/assignment3.go
package main

func SolveA3(A []int, B int, C int) []int {
	index := B
	jIndex := C

	for index < jIndex {
		A[index], A[jIndex] = A[jIndex], A[index]
		index++
		jIndex--
	}

	return A
}
